pkgs/applications/science/electronics/linux-gpib/kernel.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchurl
+, kernel
+, pahole
+}:
+
+stdenv.mkDerivation rec {
+  pname = "linux-gpib-kernel";
+  version = "4.3.6";
+
+  src = fetchurl {
```

Comment:
it would be good to reduce duplication, eg version, src. unpackPhase , while current approach is pretty clean, during next update ideally only `version` and `hash` needs to be updated once 



there are many ways of achieving it, I can suggest some

Reply:
Your suggestions are definitely welcome

Reply:
here is one of possible approaches
https://github.com/kirillrdy/nixpkgs/commit/ae82243b4b24200ce643ce9c3ecc6330348249a4

Reply:
Updated with your suggestion. Thank you.